scripts/feature_tests/upgrade/upgrade_common.sh

```diff
@@ -376,31 +368,31 @@ function create_clusterctl_configuration() {
 cat <<EOF >/home/"${USER}"/.cluster-api/clusterctl.yaml
 providers:
   - name: cluster-api
-    url: /home/$USER/.cluster-api/dev-repository/cluster-api/v0.3.5/core-components.yaml
```

Comment:
General comment: don't change the clusterctl versions and paths. The master branch content is correct.